src/main/java/com/yolo/customer/idea/IdeaService.java

```diff
@@ -33,6 +33,9 @@ public class IdeaService {
     @Autowired
```

Comment:
be consistent in dependency injection either use @AutoWired or constructor refer to already merged code